cli/main.c

```diff
@@ -426,9 +428,10 @@ static void print_bw(const char *msg, uint64_t time_us, uint64_t bytes)
 	printf("\t%-8s\t%5.3g %sB/s\n", msg, rate, suf);
 }
 
+#define CMD_DESC_BW "measure the bandwidth for each port"
```

Comment:
"measure the traffic bandwidth through each port" ??

Reply:
Sure, I'll update it to that.